bladex/profilebase.py

```diff
@@ -119,82 +129,206 @@ def interpolate_coordinates(self, num=500, radius=1.0):
+        """
+        Computes the 2D coordinates of the chord line. Also updates
+        the chord_line class member.
+
+        The chord line is the straight line that joins between the leading edge
+        and the trailing edge. It is simply computed from the equation of
+        a line passing through two points, the LE and TE.
+
+        :param bool lin_spaced: if True, then the coordinates are computed at
+            equally spaced intervals within the range of LE and TE. Default
+            value is False
+        :param int num: number of points to be used for the equally-spaced
+            sample computations, and is used only if parameter lin_spaced
+            is True. Default value is 500
+        """
+        self._update_edges()
+        aratio = (self.trailing_edge[1]-self.leading_edge[1]) / \
+                 (self.trailing_edge[0]-self.leading_edge[0])
+        if (lin_spaced==True) or \
+            (self.xup_coordinates == self.xdown_coordinates).all() == False:
+            cl_x_coordinates = np.linspace(self.leading_edge[0], \
+                                           self.trailing_edge[0], num=num)
+            cl_y_coordinates = np.zeros(cl_x_coordinates.size)
+            cl_y_coordinates = aratio \
+                               * (cl_x_coordinates - self.leading_edge[0]) \
+                               + self.leading_edge[1]
+            self.chord_line = np.array([cl_x_coordinates, cl_y_coordinates])
+        else:
+            cl_y_coordinates = np.zeros(self.xup_coordinates.size)
+            cl_y_coordinates = aratio * \
+                               (self.xup_coordinates - self.leading_edge[0]) \
+                               + self.leading_edge[1]
+            self.chord_line = np.array([self.xup_coordinates, 
+                                       cl_y_coordinates])
+
+    def get_camber_line(self, interpolate=False, n_interpolated_points=500):
+        """
+        Computes the 2D coordinates of the camber line. Also updates the
+        camber_line class member.
+
+        The camber line is defined by the curve passing through all the mid
+        points between the upper surface and the lower surface of the airfoil.
+
+        :param bool interpolate: if True, the interpolated coordinates are used
+            to obtain the camber line, otherwise the original discrete
+            coordinates are used. Default value is False.
+        :param int n_interpolated_points: number of points to be used for the
+            uniform interpolation, and is used only if parameter interpolate
+            is True. Default value is 500
+
+        We note that a uniform interpolation becomes necessary for the cases
+        when the X-coordinates of the upper and lower surfaces do not
+        correspond to the same vertical sections, since this would imply
+        inaccurate measurements for obtaining the camberline.
+        """
+        if (interpolate == True) or \
```

Comment:
Don't use backslash for line continuation [PEP 3125](https://www.python.org/dev/peps/pep-3125/).